src/cookbooks/new_ping.md

```diff
@@ -3,189 +3,247 @@
 Got some new data you want to send to us? How in the world do you send a new ping? Follow this guide
 to find out.
 
-Write Your Questions
---------------------
+Most new data collection in Firefox does not require creating a new ping document type. To add a
+histogram, scalar, or event collection to Firefox, please see the documentation on [adding a new
+probe](https://firefox-source-docs.mozilla.org/toolkit/components/telemetry/start/adding-a-new-probe.html).
+
+## Write Your Questions
+
 Do not try and implement new pings unless you know specifically what questions you're trying to
 answer. General questions about "How do users use our product?" won't cut it - these need to be
 specific, concrete asks that can be translated to data points. This will also make it easier down
 the line as you start data review.
 
-More detail on how to design and implement new pings for Firefox Desktop [can be found here](https://firefox-source-docs.mozilla.org/toolkit/components/telemetry/telemetry/collection/custom-pings.html).
+More detail on how to design and implement new pings for Firefox Desktop [can be found
+here](https://firefox-source-docs.mozilla.org/toolkit/components/telemetry/telemetry/collection/custom-pings.html).
+
+## Choose a Namespace and DocType
+
+For new telemetry pings, the namespace is simply `telemetry`. For non-Telemetry pings, choose a
```

Comment:
I believe this needs more scoping now. IMHO we shouldn't have any more custom pings come in from the `telemetry` namespace, except those added by the Firefox Client Telemetry team.